Dockerfile

```diff
@@ -1,10 +1,16 @@
 FROM alpine:3.10
 
+LABEL org.opencontainers.image.source=https://github.com/fqx/podsync-with-yt-dlp
+
 WORKDIR /app/
-RUN wget -O /usr/bin/youtube-dl https://github.com/ytdl-org/youtube-dl/releases/latest/download/youtube-dl && \
+RUN wget -O /usr/bin/youtube-dl https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp && \
     chmod +x /usr/bin/youtube-dl && \
-    apk --no-cache add ca-certificates python ffmpeg tzdata
-COPY podsync /app/podsync
+    wget -O /app/podsync.tar.gz https://github.com/mxpv/podsync/releases/download/v2.4.0/Podsync_2.4.0_Linux_x86_64.tar.gz && \
```

Comment:
This will not work. See here https://github.com/mxpv/podsync/blob/26f4b2a47af01af006acbed8133c38947641f7ad/Dockerfile#L1